### PR TITLE
Fix: allow null extra time

### PR DESCRIPTION
**Pull request by wazelin**
This PR aims to allow skipping extra-time on ACS control calls.

### CODE REVIEW

File: src/Action/Tool/Ajax/AcsServiceClientAction.php [hunk 1]
```diff
@@ -63,10 +63,10 @@ public function __invoke(Request $request): Response
             $request->get('acsSub'),
             $request->get('acsAction'),
             $date,
-            (int)$request->get('acsAttemptNumber'),
+            (int)$request->request->has('acsAttemptNumber') ?: 1,
```
Comment (kilatib) [inline src/Action/Tool/Ajax/AcsServiceClientAction.php:66]: will it not be always `1`?

Comment (wazelin) [reply src/Action/Tool/Ajax/AcsServiceClientAction.php]: Right. Forgot to change that line before committing.
Thanks, fixed.

**Review by poyuki — APPROVED** [review]
LGTM

**Review by kilatib — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [ ] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Pull request merged**